openmls/src/group/mls_group/commit_builder.rs

```diff
@@ -0,0 +1,795 @@
+//! This module contains the types for building commits.
+//!
+//! A living design doc can be found here: <https://md.cryspen.com/s/TqZXcU-gA>
```

Comment:
The docs here seem a bit draft-y. Since this is public, it would be nice to have proper documentation here.

Reply:
I guess that's because they _are_ a draft :sweat_smile:. For now I removed the drafty part and just refer to the CommitBuilder struct docs. I guess the right thing to do would be to add proper examples to the book code, but I am not sure if I should replace all the current examples with a commit builder. Happy to hear opinions.